use_cases/eluc/prescriptors/nsga2/train_prescriptors.py

```diff
@@ -1,3 +1,4 @@
+import argparse
```

Comment:
This is to enable argument passing for the train prescriptors script in the future